x/posts/internal/types/strings.go

```diff
@@ -0,0 +1,39 @@
+package types
+
+// Strings represents a slice of strings.
+type Strings []string
```

Comment:
Where is this needed?

Reply:
I implemented this at the beginning but it result useless and forgot about it, I will remove it.